lms/djangoapps/support/tests/test_views.py

```diff
@@ -406,7 +439,7 @@ def test_change_enrollment_bad_data(self, data, error_message):
         # assign the course ID here
         if 'course_id' in data and data['course_id'] is None:
             data['course_id'] = str(self.course.id)
-        response = self.client.post(self.url, data)
+        response = self.client.patch(self.url, data, content_type='application/json')
```

Comment:
why isn't data dumped here? Why is the content type header being applied explicitly?